problem-spring-web-autoconfigure/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.zalando</groupId>
+        <artifactId>problem-spring-parent</artifactId>
+        <version>0.25.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>problem-spring-web-autoconfigure</artifactId>
+
+    <name>Problem: Spring Web Autoconfiguration</name>
+    <description>Spring autoconfiguration module for Problem Spring Web</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure-processor</artifactId>
+            <version>${spring-boot.version}</version>
+            <optional>true</optional>
```

Comment:
Idiomatic would be to use `<scope>provided</scope>`, but the effect is pretty much the same.